extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -95,6 +95,7 @@ describes.realWin(
       resetServiceForTesting(win, 'consentStateManager');
       registerServiceBuilder(win, 'consentStateManager', function () {
         return Promise.resolve({
+          consentPageViewId64: () => 'foo_consent_page_view_id_64_123',
```

Comment:
nit: Let's return a Promise to match the implementation type.
```suggestion
          consentPageViewId64: () =>
            Promise.resolve('foo_consent_page_view_id_64_123'),
```